### PR TITLE
update(README): steps to add monitor and executor image added

### PR DESCRIPTION
**Pull request by shubhamchaudhary**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

**What this PR does / why we need it**:

-  update readme - add Steps to get executor and monitor image from the engine spec

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [x] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: executor/README.md [hunk 1]
```diff
@@ -89,6 +89,38 @@
             - name: FORCE
               value: "true"
     ```
+    
+### How to Override Executor and Exporter image
+
+-   To override the executor and exporter image variables , perform one of the below steps before applying the manifest:
```
Comment (ksatchit) [inline executor/README.md]: Something in the lines of: 

To provide a chaos executor (runner) & exporter (monitor) of choice, provide the image names (in the form `registry/user/image:tag`) in the chaosEngine as shown below. The defaults at this point correspond to the litmuschaos CI images in dockerhub:

- Executor: litmuschaos/ansible-runner:ci
- Monitor: litmuschaos/chaos-exporter:ci

Comment (ksatchit) [reply executor/README.md]: let's not say `one of the below` as there may be need for both.

**Comment by chandankumar4** [comment]
@shubhamchaudhary Can you check the codacy issue

**Comment by shubhamchaudhary** [comment]
> @shubhamchaudhary Can you check the codacy issue

**Comment by shubhamchaudhary** [comment]
#901 issue resolved

**Pull request merged**